code/modules/admin/create_mob.dm

```diff
@@ -11,7 +11,7 @@
 	user << browse(create_panel_helper(create_mob_html), "window=create_mob;size=425x475")
 
 /proc/randomize_human(mob/living/carbon/human/H)
-	H.gender = pick(MALE, FEMALE)
+	H.gender = pick(MALE, FEMALE, PLURAL, NEUTER)
```

Comment:
Body model isn't setup properly to handle this, and will result in 1/4 of characters having a male body model and the rest having the female body model.